test/Experimental/JuLie/schur_polynomials.jl

```diff
@@ -13,12 +12,12 @@ import Oscar.JuLie: schur_polynomial_cbf
 	@test schur_polynomial(S, Partition([3])) == x[1]*x[2]*x[3] + x[1]^2*x[2] + x[1]*x[2]^2 + x[1]^2*x[3] + x[1]*x[3]^2 + x[2]^2*x[3] + x[2]*x[3]^2 + x[1]^3 + x[2]^3 + x[3]^3
 	@test schur_polynomial(S, Partition([])) == 1
 
-	# schur_polynomial(λ, R, n)
+	# schur_polynomial(R, λ, n)
 	@test schur_polynomial(S, Partition([1]), 3) == x[1] + x[2] + x[3]
 	@test schur_polynomial(S, Partition([1,1]), 3) == x[1]*x[2] + x[1]*x[3] + x[2]*x[3]
 	@test schur_polynomial(S, Partition([2]), 3) == x[1]^2 + x[2]^2 + x[1]*x[2] + x[1]*x[3] + x[2]*x[3] + x[3]^2
 	@test schur_polynomial(S, Partition([1,1,1]), 3) == x[1]*x[2]*x[3]
-	@test schur_polynomial(S, Partition([1,1,1]), 4) == x[1]*x[2]*x[3]
+	#@test schur_polynomial(S, Partition([1,1,1]), 4) == x[1]*x[2]*x[3]
```

Comment:
Was this test disabled on purpose? If so, why?

Reply:
Ahhh, good point, thanks for reminding me, I wanted to ask @ulthiel about this:
This is a partition of 3, with three components, and we gave a polynomial ring with three variables. Yet we asked for a Schur polynomial in four variables... which the code silently truncated to 3 variables. Which I find problematic, I'd rather have an error. But perhaps there is a rationale for this I am simply not aware. In that case we can allow this again, and extend the docstrings to describe explain this behavior.

Reply:
Okay, so, this starts from the combinatorial formula for Schur polynomials: $s_\lambda = \sum_T x^T$ where the sum is over all semistandard Young tableaux of shape $\lambda$ and $x^T = x_1^{t_1} \cdots x_n^{t_n}$, where $t_i$ is the multiplicity of $i$ in $T$. So, the number of variables you need (at minimum) equals the maximum value of the entries in T. But semistandard Tableaux are always relative to an alphabet, usually 1 up to some maximal number. But this maximal number can be chosen to be anything from 1 to infinity (see also `semistandard_tableaux` and also [Sage](https://doc.sagemath.org/html/en/reference/combinat/sage/combinat/tableau.html#sage.combinat.tableau.SemistandardTableaux)). So, the n can be any number >= 1 from the combinatorial viewpoint. If n is not specified then it should default the _length_ of $\lambda$; not the sum as in the code, which would be the default for listing semistandard tableaux, that's where this typo may come from.

Example: 
`
julia> semistandard_tableaux(Partition([3,2]), 2)
2-element Vector{Tableau{Int64}}:
 [[1, 1, 1], [2, 2]]
 [[1, 1, 2], [2, 2]]

julia> schur_polynomial(Partition([3,2]),2)
x1^3*x2^2 + x1^2*x2^3
`

Reply:
I am not entirely sure though whether the implementation of the bialternant formula works in this combinatorial generality though I have to check this.

Reply:
Yes, I think also in the bialternant formula you can take any number of variables, so this should be okay. But you are correct about `schur_polynomial(S, Partition([1,1,1]), 4)` when S is a polynomial ring in 3 variables. This should not happen. It needs to have at least n variables.